plugins/scaffolder/src/components/fields/EntityPicker/EntityPicker.tsx

```diff
@@ -151,11 +159,15 @@ export const EntityPicker = (props: EntityPickerProps) => {
         loading={loading}
         onChange={onSelect}
         options={entities || []}
+        renderOption={option => <EntityDisplayName entityRef={option} />}
```

Comment:
man now that you have a `renderOption`, is there any way that we can just remove the entire `getOptionLabel` as well? or does that ruin things

Reply:
I don't think we can remove it because the Autocomplete still needs to render the selected option (which cannot be a custom components like the options from what I saw) and to do that it needs to get the label of the selected option 🙁 

As stated in [Mui docs](https://mui.com/material-ui/api/autocomplete/#autocomplete-prop-getOptionLabel) : 

```
getOptionLabel :   
Used to determine the string value for a given option.   
It's used to fill the input (and the list box options if renderOption is not provided).
```